### PR TITLE
fix: reset node status on restart node

### LINKED ISSUE #1463: Error Message When Trying to Stop Smeshing During Syncing
### Environment

- Smapp version: latest

### Describe the Bug

Currently, there is an issue when the smeshing process or node syncing is ongoing, and the user attempts to stop smeshing. On the Smeshing screen, clicking the "Stop Smeshing" button displays the following message:

Screenshot

### Steps to Reproduce

This behavior is not consistent and depends on the node's status.

1. Open Smapp with smeshing enabled.
2. Quickly navigate to the Smeshing screen.
3. Click "Stop Smeshing".

### Expected Behavior

All scenarios where stopping smeshing is not possible should be appropriately handled. If unfeasible, the "Stop Smeshing" button should not be displayed. Alternatively, if an error message appears, it should clarify that this is an expected behavior and encourage the user to try again later.

### Actual Behavior

After clicking the button, an error modal is shown. The user closes it and can attempt the action again. The outcome might still lead to an error message (depending on the situation).

### PR DESCRIPTION
**Pull request by maparr**
It closes #1463

### CODE REVIEW

File: desktop/NodeManager.ts [hunk 1]
```diff
@@ -300,7 +303,7 @@ class NodeManager extends AbstractManager {
     if (this.isNodeRunning()) return true;
     this.$_nodeStatus.reset();
     await this.spawnNode();
-    this.startGRPCClient();
+    await this.startGRPCClient();
```
Comment (maparr) [inline desktop/NodeManager.ts]: till node is not connected on login process, we will be showing loader

File: desktop/NodeManager.ts [hunk 2]
```diff
@@ -278,6 +278,9 @@ class NodeManager extends AbstractManager {
 
   startGRPCClient = async () => {
     this.nodeService.createService();
+
+    this.sendNodeStatus(DEFAULT_NODE_STATUS);
```
Comment (maparr) [inline desktop/NodeManager.ts]: reset previous stale node status on restart node

File: desktop/NodeManager.ts (hunk 1 above)
Comment (maparr) [reply desktop/NodeManager.ts]: reverted, for not powerful PC's , this screen may be a blocker

**Comment by brusherru** [comment]
I've rebased it

**Pull request merged**